docs/_toc.yml

```
@@ -5,4 +5,5 @@ chapters:
   - file: tutorials/diafiltration
   - file: tutorials/diafiltration-solution
   - file: tutorials/uky_flowsheet-solution
+  - file: tutorials/byproduct_recovery_determination
```

Rename the file so that the CI framework can find it:

```suggestion
  - file: tutorials/byproduct_recovery_determination-solution
```